### ISSUE TITLE
CreateBucket call had errors: [SignatureDoesNotMatch] - while using with s3 on Linode

### ISSUE THREAD

**Issue by szabgab** [leejo/AWS-S3#17]
I am trying to use the AWS:S3 module with the S3 of Linode.

The following code gives me an error:

```
CreateBucket call had errors: [SignatureDoesNotMatch]  at /home/gabor/perl5/lib/perl5/AWS/S3.pm line 108.
```

Using the same credentials in python script with boto3 works.

```
use strict;
use warnings;
use feature 'say';

use Dotenv;
use AWS::S3;

Dotenv->load;

my $url = "diggers.us-southeast-1.linodeobjects.com";


my $s3 = AWS::S3->new(
  access_key_id     => $ENV{AWS_ACCESS_KEY_ID},
  secret_access_key => $ENV{AWS_SECRET_ACCESS_KEY},
  endpoint => $url,
);

my $bucket = $s3->add_bucket(
  name    => 'qqrq',
);

```

If instead of `add_bucket` I call `say $s3->buckets;` it does not print anything despite already having buckets.

ps. I'd be happy to send you temporary credentials to try it if that helps.

**Comment by leejo** [comment]
Yeah, this is possibly due to #13 or #7, or something else. Sorry @szabgab I have no time to look into this, and we don't use the dist in our stack anymore. I'm happy to review and accept a patch however, and of course upload to CPAN. I'm only co-maint for this dist on CPAN so can't give out additional co-maint on it either (JOHND has first-come).

**Comment by szabgab** [comment]
Thanks. I might look into this though in the meantime I think I found other solutions for my needs.

**Comment by leejo** [comment]
I've just pushed v1.00 to CPAN, which should resolve this issue.
